modules/gdscript/gdscript_editor.cpp

```diff
@@ -1951,7 +1951,7 @@ static void _find_identifiers_in_base(const GDScriptCompletionContext &p_context
 					} else {
 						base_type.has_type = script->get_instance_base_type() != StringName();
 						base_type.kind = GDScriptParser::DataType::NATIVE;
-						base_type.script_type = script->get_instance_base_type();
```

Comment:
The glories of static typing.. and auto-coercion.

Note that `get_instance_base_type` returns `StringName`, while `script_type` is a `Ref<Script>`. How C++ manages to convert one into the other is a complete mystery to me. 
Explanations are welcome!